Cargo.toml

```diff
@@ -34,3 +34,7 @@ cli = { path = "rust/cli" }
 [features]
 release = ["cli/release"]
 scenario_runner = ["cli/scenario_runner"]
+
+[patch.crates-io]
+# NOTE https://github.com/ipetkov/crane/issues/336
+libffi-sys = { git = "https://github.com/tov/libffi-rs", rev = "d0704d634b6f3ffef5b6fc7e07fe965a1cff5c7b" }
```

Comment:
Created https://github.com/tov/libffi-rs/issues/98 to be able to remove this patch at some point